pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -53,7 +53,13 @@ stdenv.mkDerivation rec {
 
     dontFixup = true;
     outputHashMode = "recursive";
-    outputHash = "sha256-KDJ8QmpwGb2lOdwWEl5y62pJiqEvpI59StfQZrN1PPE=";
+    outputHash =
+      let
+        inherit (stdenv.hostPlatform) system;
+      in
+      if system == "aarch64-linux" then
```

Comment:
Nit: Can't we just have `stdenv.hostPlatform.system` here? That let-in feels kind of unnecessary

Reply:
i used the let in so that it would be easier to add more systems in the future like `i686-linux` or darwin; it's pointless now, but hopefully it'll help later